<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>switchablegraphics</class>
 <widget class="QWidget" name="switchablegraphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="autoSwitch">
     <property name="title">
      <string>Enable Automatic Switch</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="lowBatSwitchFlag">
        <property name="text">
         <string>Enable switch on low battery</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="perc_layout">
        <item>
         <widget class="QSlider" name="sliderPerc">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="inputPerc">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="statusIconFlag">
     <property name="title">
      <string>Enable Status Icon</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="notificationsFlag">
        <property name="text">
         <string>Enable Notifications</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Info</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="infoLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderPerc</sender>
   <signal>valueChanged(int)</signal>
   <receiver>inputPerc</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lowBatSwitchFlag</sender>
   <signal>toggled(bool)</signal>
   <receiver>sliderPerc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lowBatSwitchFlag</sender>
   <signal>toggled(bool)</signal>
   <receiver>inputPerc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputPerc</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderPerc</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
